integration_tests/integration/incidentDetailsSubmittedEdit.cy.ts

```diff
@@ -273,10 +281,95 @@ context('Incident details (edit after completion of report)', () => {
       incidentDetailsPage.timeInputHours().type('13')
       incidentDetailsPage.timeInputMinutes().clear()
       incidentDetailsPage.timeInputMinutes().type('00')
+      incidentDetailsPage.radioButtonsDiscovery().find('input[value="Yes"]').click()
+      incidentDetailsPage.submitButton().click()
+      cy.location().should(loc => {
+        expect(loc.pathname).to.eq(`${adjudicationUrls.detailsOfOffence.urls.start(34)}`)
+      })
+    })
+
+    it('should show no appropraite hour error if DISCOVERY radio is  selected and hour is wrong', () => {
+      cy.visit(
+        `${adjudicationUrls.incidentDetails.urls.submittedEdit(
+          'G6415GD',
+          34
+        )}?referrer=${adjudicationUrls.prisonerReport.urls.report(1524455)}`
+      )
+      const incidentDetailsPage: IncidentDetails = Page.verifyOnPage(IncidentDetails)
+      incidentDetailsPage.radioButtonsDiscovery().find('input[value="No"]').click()
+      incidentDetailsPage.timeInputMinutes().clear()
+      incidentDetailsPage.timeInputMinutes().type('13')
```

Comment:
If your test is about changing the time of discovery, I think changing the incident time too overcomplicates the test (because you aren't asserting anything about it anyway). So these two lines aren't required